api/GlobalEventHandlers.json

```diff
@@ -204,10 +204,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": "9"
```

Comment:
Was `onanimationend` perhaps supported on `Element` or `HTMLElement` in Safari 9? Same for changes below.

Reply:
Interestingly enough, it wasn't supported on `Document`, but it was supported on all other interfaces.  Reverting these changes.